src/Formplot/FileFormat/FormplotReader.cs

```diff
@@ -56,9 +57,32 @@ internal static class FormplotReader
 			var version = ReadFileVersion( zipFile );
 			var metadataStream = ReadMetadataStream( zipFile );
 
-			using var pointDataStream = zipFile.GetEntry( "plotpoints.dat" )?.Open();
+			var zipEntry = zipFile.GetEntry( "plotpoints.dat" );
+			if( zipEntry is null )
```

Comment:
A comment right here explaining why this branch might be taken (name-value pairs) might be good for our future selfs.